solutions/security/detect-and-alert/suppress-detection-alerts.md

```diff
@@ -42,45 +26,46 @@ Alert suppression is not available for Elastic prebuilt rules. However, if you w
 
```

Comment:
Suppression for prebuilt rules can be configured from 8.18 on Enterprise license when rules are edited. Please change this note. It's also good to highlight that that is possible.

Reply:
Good catch - note updated in [3ff6318](https://github.com/elastic/docs-content/pull/2735/commits/3ff63180d70fbe95770d21eddd5340b29664a0ca).